vlasiator.cpp

```diff
@@ -590,7 +590,7 @@ int main(int argn,char* args[]) {
             writeGhosts
          ) == false
       ) {
-         cerr << "FAILED TO WRITE GRID AT " << __FILE__ << " " << __LINE__ << endl;
+         abort_mpi("FAILED TO WRITE GRID", 1);
```

Comment:
This and later: I would greatly prefer to have the file name and line number passed down to abort_mpi and then error type 0.